index.d.ts

```diff
@@ -0,0 +1,28 @@
+declare module 'lazydecipheriv' {
+  import * as stream from 'stream';
+  import {
+    CipherCCMTypes,
+    BinaryLike,
+    CipherCCMOptions,
+    DecipherCCM,
```

Comment:
i would just make the single file ts rather than adding a separate types file

Reply:
Since this is open source module to be used instead of `crypto.createDecipheriv`, it's best to match it to the JavaScript that Node is in.